src/materials/ConcreteExpansionEigenstrainBase.C

```diff
@@ -184,16 +185,12 @@ ConcreteExpansionEigenstrainBase::computeAB(const Real ab1,
                                             const Real ab3,
                                             const unsigned int pbound)
 {
-  Real ab;
   if (pbound == 0)
```

Comment:
I'd put an assert at the beginning of this function:
```
mooseAssert("pbound <= 2", "pbound outside allowed range");
```
Since `pbound` is unsigned, you don't need to check for the negative case in the assertion.

Reply:
I will address this too.